### PR TITLE
Edit level scripts suggested in tutorial meetings

### PR DESCRIPTION
**Pull request by hollan2**

### CODE REVIEW

File: Isle of Trust/src/routes/TutorialGameView.tsx [hunk 1]
```diff
@@ -596,46 +594,56 @@ class TutorialView extends React.Component<StartInfo, GameViewState> {
...
+                );
+            }
+            // Not all players can pay 5 ton mortgage
+            else {
+                return (
+                    <EndOfLevel
+                        level={this.props.level}
+                        success={false}
+                        mapAgents={checkResource}
+                    />
```
Comment (ViktoriyaPetrova) [inline Isle of Trust/src/routes/TutorialGameView.tsx:644]: Change line 642 to` success={true}` to test the end of level pop up for level 7 without actually having to win the level

**Review by hollan2 — COMMENTED** [review]
Level 6 - Have Susi popup be round 5 or 6.

**Review by hollan2 — COMMENTED** [review]
all the tutorial levels seem functioning and winnable to me :) except for level 7. We might want to increase Profe's resources to 13 -15?

**Review by Cpod2 — APPROVED** [review]
looks good to me!

**Pull request merged**